### PR TITLE
Nav bar UI fix

### PR DESCRIPTION
**Pull request by JCSime**
## Description

- Removes ‘Account’ from sidebar
- Adds an alt and aria-label to the far right of nav bar that says ‘Account settings’ for accessibility readers
- Adds icons to ‘Account settings’ nav bar dropdown menu
- Removes unnecessary ‘Memo’ dropdown menu from nav bar
- Changes the color of the nav ‘Memo’ button to white for accessibility reasons

#### Video Link

[Loom Video] https://www.loom.com/share/6fc8faefec264f638286b636d94bfacd

#### Trello Link

<blockquote class="trello-card"><a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;uUKuzW1H&#x2F;311-as-a-user-the-ui-features-on-the-sidebar-and-nav-bar-are-attractive-and-easy-to-find">As a user the UI features on the sidebar and nav bar are attractive and easy to find</a></blockquote><script src="https://p.trellocdn.com/embed.min.js"></script>

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

### CODE REVIEW

**Comment by aaronamendez** [comment]
Looks good guys, great job as a coding pair!

**Review by Jmz0127 — COMMENTED** [review]
Love the addition of screen readers to the menu bar to boost accessibility. way to go!

**Comment by Collyngodlewski** [comment]
Nice!! The dropdown looks a lot cleaner there and nice you added the screen reader aspect. Wonderful job guys

File: src/components/pages/Navbar/Navbar.js [hunk 1]
```diff
@@ -82,22 +67,34 @@ const Navbar = ({ isAuthenticated, userProfile }) => {
...
+                  Memos
+                </Link>
+                <Dropdown overlay={accountMenu} placement="bottom" arrow>
                   <div className="userInfo-and-profilePic">
                     <div className="profilePic">
                       <Avatar
                         size={50}
                         icon={<UserOutlined />}
                         src={profilePic}
+                        alt="Account settings"
```
Comment (kimnivore) [inline src/components/pages/Navbar/Navbar.js:88]: I like how you added the alt tags for screen readers. Thanks for demonstrating this in the video. It was really helpful seeing it in action.

**Review by kimnivore — APPROVED** [review]
Your code was easy to understand. I like the addition of the alt tags for screen readers. Great job!

**Review by BrittanyPete — CHANGES_REQUESTED** [review]
It looks like there are a few things that are defined but never used. Could you help clear some of that up? Examples: Link, Modal, UserOutlined, LogoutOutlined, modal, openModal, cancelOpen, handleLogout, axiosWithAuth, useForms. Other than that, I really like how you guys incorporated the alt tags for screen readers!

**Review by xokeixo — COMMENTED** [review]
Everything looks great! Very readable and accessible.

**Comment by JCSime** [comment]
> It looks like there are a few things that are defined but never used. Could you help clear some of that up? Examples: Link, Modal, UserOutlined, LogoutOutlined, modal, openModal, cancelOpen, handleLogout, axiosWithAuth, useForms. Other than that, I really like how you guys incorporated the alt tags for screen readers!

Just pushed those changes, thanks for giving me the heads up about those! Forgot to clean that up after Ryan mentioned it in the meeting.

**Pull request merged**